Assets/Scripts/Gameplay/GameplayObjects/ClientTossedItem.cs

```diff
@@ -0,0 +1,16 @@
+using Unity.Netcode;
+using UnityEngine;
+using UnityEngine.Events;
+
+namespace Unity.Multiplayer.Samples.BossRoom.Client
+{
+    public class ClientTossedItem : NetworkBehaviour
```

Comment:
So it's like we discussed the other day, we don't _need_ client/server separation everywhere, right. Here this separation is a bit trivial, no? Could this be merged with the ServerTossedItem?
Should we document client/server separation patterns and how to apply them? Could be interesting to create a small guide in our doc for what we think and how this applies to boss room, do you want to look into it?